### PR TITLE
Fix multiple issues with API calls

### PR DESCRIPTION
**Pull request by Syzuna**
While adding the XML docs I noticed some API calls that had missing or removed query params.
I am sure I still missed a lot of them but its a start.

### CODE REVIEW

File: TwitchLib.Api.Helix/Analytics.cs [hunk 1]
```diff
@@ -27,15 +29,54 @@ public Analytics(IApiSettings settings, IRateLimiter rateLimiter, IHttpCallHandl
...
+                new KeyValuePair<string, string>("first", first.ToString())
+            };
 
             if (!string.IsNullOrWhiteSpace(gameId))
                 getParams.Add(new KeyValuePair<string, string>("game_id", gameId));
 
+            if (startedAt != null && endedAt != null)
+            {
+                getParams.Add(new KeyValuePair<string, string>("started_at", startedAt.Value.ToRfc3339String()));
+                getParams.Add(new KeyValuePair<string, string>("started_at", endedAt.Value.ToRfc3339String()));
```
Comment (swiftyspiffy) [inline TwitchLib.Api.Helix/Analytics.cs:71]: `ended_at` right?

File: TwitchLib.Api.Helix/Analytics.cs [hunk 2]
```diff
@@ -52,15 +93,54 @@ public Task<GetGameAnalyticsResponse> GetGameAnalyticsAsync(string gameId = null
...
+                new KeyValuePair<string, string>("first", first.ToString())
+            };
 
             if (!string.IsNullOrWhiteSpace(extensionId))
                 getParams.Add(new KeyValuePair<string, string>("extension_id", extensionId));
 
+            if (startedAt != null && endedAt != null)
+            {
+                getParams.Add(new KeyValuePair<string, string>("started_at", startedAt.Value.ToRfc3339String()));
+                getParams.Add(new KeyValuePair<string, string>("started_at", endedAt.Value.ToRfc3339String()));
```
Comment (swiftyspiffy) [inline TwitchLib.Api.Helix/Analytics.cs:135]: `ended_at`

**Review by swiftyspiffy — CHANGES_REQUESTED** [review]

File: TwitchLib.Api.Helix/Analytics.cs (hunk 2 above)
Comment (Syzuna) [reply TwitchLib.Api.Helix/Analytics.cs:135]: yeah copy paste error.
will fix that

**Review by swiftyspiffy — APPROVED** [review]
lgtm

**Pull request merged**